src/common/types/store-data/unified-data-interface.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// this is similar to `TestEngine` interface from axe-core
+export interface ScanEngine {
+    name: string;
+    version: string;
+}
+
+export interface OSData {
+    name: string;
+    osVersion?: string;
+    userAgent?: string;
+}
+
+export interface ViewPortData {
```

Comment:
Could we change this to something simple like ScreenProperties: { height: number, width: number}?

Reply:
We weren't sure if that made sense for web (and maybe axe-windows), since our "viewport" would actually be a "window." It makes more sense for axe-android to have "ScreenProperties."

Reply:
I will remove "platform" from "platformHeight" and "platformWidth" for now.